src/app/api/crud.py

```diff
@@ -43,7 +43,7 @@ async def put(entry_id: int, payload: BaseModel, table: Table) -> int:
 
 async def delete(entry_id: int, table: Table):
     query = table.delete().where(entry_id == table.c.id)
-    return await database.execute(query=query)
+    await database.execute(query=query)
```

Comment:
This does the output, could you add the typing `-> None` for this function then?

Reply:
The delete execution was already not producing any return value. I can explicitly note it

Reply:
Yes please :pray:  (I remember myself trying to figure out what it was returning, and didn't find anything)

Reply:
I tried to make the query returning the deleted entry to avoid doing 2 db operations (get + delete) but it seams to be impossible

Reply:
What do you mean? it's alright not returning anything, I was just asking to add typing `-> None` to the return type of the function